patches/chrome-browser-ui-browser_navigator.cc.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
+index 57cc96c276c1106079828b5e359a3d88e1c901ce..aa184ea5a2881ebbce3a106e55ac864212329ce6 100644
+--- a/chrome/browser/ui/browser_navigator.cc
++++ b/chrome/browser/ui/browser_navigator.cc
+@@ -707,6 +707,11 @@ bool IsHostAllowedInIncognito(const GURL& url) {
+            host != chrome::kChromeUISuggestionsHost;
+   }
+ 
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
why is this needed?

Reply:
This is needed for our sync page.
Our sync page is not allowed to load private window.
That causes crash for now.

Reply:
so this is a good example of what I brought up in slack today. If we create this one-off patch then someone else will have to come in and create another one if there is some other host we don't want in private tabs. We should have a chromium_src override that defines a method to call here